packages/next-app/src/app/stableswapsimulator/(components)/CustomDataForm.tsx

```diff
@@ -46,13 +47,19 @@ export default function CustomDataForm() {
     if (Object.keys(errors).length) return;
 
     const data = getValues();
-    setCustomData(data as AnalysisData);
+    const dataToCalculate = {
+      ...data,
+      swapFee: percentToNumber(data.swapFee),
+    };
+    setCustomData(dataToCalculate as AnalysisData);
```

Comment:
Think that would be easier to let this number as percent and transform it just on the math interface.

Reply:
yeah, it was clear to me that this issue has different approaches, but only changing on math we would have still to transform the swapFee when importing a pool, as the data from to subgraph comes as the full number. 

Think our suggested approach is very valid, but at first, wanted to do both transformations in the same file